File: TuristickaAgencija.Model/Smjestaj.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public class Smjestaj
    {
        public int Id { get; set; }

        public string NazivSmjestaja { get; set; }
        public string OpisSmjestaja { get; set; }
      
        public float CijenaNocenja { get; set; }

        public string Tip_sobe { get; set; }

        public byte[] Slika { get; set; }
    }
}

File: UI/turisticka_agencija_mobile/lib/models/Smjestaj.dart
import 'package:json_annotation/json_annotation.dart';

part 'Smjestaj.g.dart';

@JsonSerializable()
class Smjestaj {
    int cijenaNocenja;
    int id;
    String nazivSmjestaja;
    String opisSmjestaja;
    String slika;
    String tip_sobe;

    Smjestaj({required this.cijenaNocenja,required this.id,required this.nazivSmjestaja,required this.opisSmjestaja,
        required this.slika, required this.tip_sobe});

    factory Smjestaj.fromJson(Map<String, dynamic> json) => _$SmjestajFromJson(json);

    Map<String, dynamic> toJson() => _$SmjestajToJson(this);
}